web/src/routes/news/containers/LocalNewsPage.js

```diff
@@ -58,11 +52,7 @@ export const LocalNewsPage = ({
     />
   }
 
-  if (areCitiesFetching) {
-    return <LoadingSpinner />
```

Comment:
No `LoadingSpinner`?

Reply:
Cities are always loaded as they are included in the required payloads of the route, so no need to check it here again.